utils/appliance.py

```diff
@@ -1364,6 +1364,18 @@ def get_yaml_config(self, config_name):
     def set_yaml_config(self, config_name, data_dict):
         return db.set_yaml_config(config_name, data_dict, self.address)
 
+    def set_session_timeout(self, timeout=86400):
+        try:
+            vmdb_config = self.get_yaml_config("vmdb")
+            if vmdb_config["session"]["timeout"] < timeout:
+                vmdb_config["session"]["timeout"] = timeout
+                self.set_yaml_config("vmdb", vmdb_config)
+        except Exception as ex:
+            # Definitely need to implement retries or something, this is
+            # just a bandaid
+            logger.error('Setting session timout failed')
+            logger.exception(ex)
```

Comment:
This little hack I put in was tough to justify as a fixture, but in this method it's even less good...

Unfortunately, I think it's still necessary, so a good compromise would at least be to allow the caller to specify if this should raise an exception or only log it, with the default probably being the current behavior (logging it), and other usages inside tests raising exceptions.